app/helpers/application_helper.rb

```diff
@@ -54,8 +54,8 @@ def base_title
 
   def sortable(column, title = nil)
     title ||= column.titleize
-    css_class = (column == params[:sort]) ? "current #{params[:direction]}" : nil
-    direction = (column == params[:sort] && params[:direction] == 'asc') ? 'desc' : 'asc'
+    css_class = column == params[:sort] ? "current #{params[:direction]}" : nil
+    direction = column == params[:sort] && params[:direction] == 'asc' ? 'desc' : 'asc'
```

Comment:
Realizing it's completely a style thing, but I personally like parenthesis around compound conditions in ternaries. It keeps me from second guessing operator precedence, and I often find it more clear to read. Thoughts?